src/content/testing/getting-started.mdx

```diff
@@ -0,0 +1,288 @@
+---
+title: Getting started with Test Containers
+description: In this section, we'll see how you caconfigure Test Containers for your application
+sidebar_label: Getting Started
+id: getting-started
+---
+
+This guide explains how to configure Test Containers for your first app to Okteto.
+
+## Requirements
+
+Before you start this tutorial, make sure you fulfill the following requirements:
+
+- You have access to an Okteto instance. Otherwise, follow our [installation guide](get-started/install/index.mdx)
+- You have [configured the Okteto CLI](get-started/install-okteto-cli.mdx) with your Okteto instance
+- You have complited [our guide to deploy your first app to Okteto](get-started/deploy-your-app/index.mdx)
+
+## What you will be building
+
+This tutorial will guide you on how to test [Movies app](https://github.com/okteto/getting-started) using Okteto.
+
+## Step 1: Deploy the Movies App
+
+Get a local version of the Go Sample App by executing the following commands:
```

Comment:
Movies maybe?
```suggestion
Get a local version of the Movies Sample App by executing the following commands:
```

Reply:
Yes, in the current docs we mostly use "Movies app" for the other section of the getting started. cc. @pchico83